src/modules/systemlib/print_load_posix.c

```diff
@@ -91,15 +91,10 @@ void print_load(uint64_t t, int fd, struct print_load_s *print_state)
 		clear_line = CL;
 	}
 
-#if defined (__PX4_LINUX)
-	dprintf(fd, "%sTOP NOT IMPLEMENTED ON LINUX\n",
-		clear_line);
+#if defined(__PX4_LINUX) || defined(__PX4_CYGWIN) || defined(__PX4_QURT)
+	dprintf(fd, "%sTOP NOT IMPLEMENTED ON LINUX, QURT, WINDOWS (ONLY ON APPLE)\n", clear_line);
```

Comment:
Nitpick: it's implemented on NuttX too.

Reply:
Oh yeah, obviously you're right. I was thinking in the posix box but the message might be misleading.